CSV accession import event support, refs #13315

mcantelon opened the pull request:
Adds functionality, to CSV accession import, to allow accession events
to be imported.

mcantelon committed:
Add accession events, refs #13315

This adds support for accession events, based on the Canadian Archival
Accession Information Standard (CAAIS).

* Add new AtoM object type: accession events
* Add accession event type taxonomy
* Add "Physical transfer" accession event type term
* Add method to accession model to get an accession's accession events
* Add ability to add/edit accession events via the web UI
* Add Elasticsearch document type for accession events
* Update accession search/indexing to incorporate accession event
  types, dates, agents, and notes

mcantelon committed:
CSV accession import event support, refs #13315

Adds functionality, to CSV accession import, to allow accession events
to be imported.

djjuhasz committed:
Remove hard cap on nodeLimit, refs #13320

Remove the hard cap of 1000 on the nodeLimit parameter for the full width
treeview and hierarchy browser

mcantelon commented:
(This will need to be merged after the AtoM 2.6.x release and after the other accession events work is merged.)

sbreker reviewed (approved):
Looks good @mcantelon! 👍

mcantelon committed:
Fixed accession edit markup, refs #13315

Fixed minor issue with markup that was preventing tooltips from working
properly.

mcantelon commented:
Cherry picked to qa/2.x
